src/record/record.js

```diff
@@ -449,7 +446,16 @@ Record.prototype._applyUpdate = function( message ) {
  */
 Record.prototype._onRead = function( message ) {
 	this.version = parseInt( message.data[ 1 ], 10 );
-	this._applyChange( jsonPath.set( this._$data, undefined, JSON.parse( message.data[ 2 ] ) ) );
+
+	this._$data = JSON.parse( message.data[ 2 ] );
```

Comment:
Okay, so:

###### Old behavior - record is/isn't ready - triggerNow true:
1) Subscribe to path `name`
2) Get read -> {}
3) name triggered with undefined


###### Old behavior - record is/isn't ready - triggerNow true:
1) Subscribe to path `name`
2) Get read -> {}
3) name triggered with undefined
###### New behavior - record is ready:
1) Subscribe to path `name`
2) Nothing..

###### New behavior - record isn't ready:
1) Subscribe to path `name`
2) Get read -> {}
3) name triggered with undefined